app/assets/stylesheets/layout_components/reveal-modal.scss

```diff
@@ -20,6 +20,25 @@ dialog .close-reveal-modal {
     font-weight: normal;
   }
 
+  &--chat {
+    @extend .reveal-modal;
+    position: fixed;
+    bottom: 0;
+    top: auto !important;
```

Comment:
Is the `!important` necessary here? I know we need the `top: auto`, but I just ask because I can't see the auto overriding any other styles which makes me think we may not need it. I could be wrong about this one though!

Reply:
we needed the auto to override a top being applied, and it only works with the important.